src/canmatrix/formats/xls.py

```diff
@@ -114,20 +114,17 @@ def dump(db, file, **options):
                 'Signal Name', 'Signal Function', 'Signal Length [Bit]', 'Signal Default', ' Signal Not Available', 'Byteorder']
     head_tail = ['Value',   'Name / Phys. Range', 'Function / Increment Unit']
 
-    if "additionalAttributes" in options  and len(options["additionalAttributes"]) > 0:
-        additional_signal_colums = options["additionalAttributes"].split(",")
+    if len(options.get("additionalAttributes", "")) > 0:
```

Comment:
again single line `additional_signal_colums = options.get("additionalAttributes", "").split(",")`  (This is also more mypy friendly, otherwise the static check might require type annotation for the empty list. Just side note.)